### PR TITLE
Created home page

### LINKED ISSUE #4: Created home page
Created search bar component
Created notepad component
Created add new note button
Made responsive

### PR DESCRIPTION
**Pull request by Raghav888**
1. Created search bar component
2. Created notepad component
3. Created add new note button
4. Made responsive
5. https://deploy-preview-3--mynoteapps.netlify.app/

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
- This code looks fine to me.
- Clean code.
- hierarchy is maintained.
- This PR is good to merge.

**Review by ShreyashMahajan — APPROVED** [review]
UI looks very clean and I liked its simplicity so much. 
Code looks structured and understandable.
Each and everything is broken into small components.
Good to merge

File: src/components/Notepad/Notepad.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import "./notepad.css";
+import React from "react";
+export const Notepad = (props) => {
```
Comment (sushil-mod) [inline src/components/Notepad/Notepad.js:3]: this is also correct , if possible you can try  direct "{ item }" instead of props

**Review by sushil-mod — APPROVED** [review]
just a suggestion .overall code is nice

**Pull request merged**